src/app/pages/dynamic-stepper/dynamic-stepper.component.scss

```diff
@@ -0,0 +1,43 @@
+@import '~@pxblue/colors/palette.scss';
+
+.radio-group {
+    display: flex;
+    flex-direction: column;
+}
+
+.delete-icon {
+    vertical-align: middle;
+    color: map-get($pxb-black, 500);
+    font-size: 18px;
+    line-height: 24px;
+}
+
+.add-button {
+    width: 100%;
+    height: 100%;
+    padding: 16px 64px;
```

Comment:
This is going to sound very nitpicky, but the text in the Add a step row and the numbers steps are off by 4 pixels (64 vs 60). If you change this from 64px to 60, they'll line up...it'll be easier than changing the margin on the steps bc i think that's coming from theme styles.